src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        ColorSupplier supplier = new ColorSupplier();
+        int number = random.nextInt(100);
+        String color = String.valueOf(supplier.getRandomColor());
```

Comment:
Instead of using `String.valueOf()` to get the string representation of the enum constant, you should use the `name()` method. This ensures that even if the `toString()` method is overridden in the enum, the actual name of the enum constant is returned.